wdl/AnnotateILFeatures.wdl

```diff
@@ -225,7 +225,6 @@ workflow AnnoILFeatures {
     call MiniTasks.ConcatVcfs as ConcatVcfsIL{
         input:
             vcfs=Bcf2VcfIL.vcf,
-            merge_sort=true,
```

Comment:
Looks like this is reverting a recent change?

Reply:
The task ConcatVcfs in TasksMakeCohortVcf.wdl doesn't have that argument in master.  Is there some other branch that isn't checked in yet that I ought to be basing this on?